RInAppMessaging/Classes/Views/FullScreenView.swift

```diff
@@ -1,6 +1,14 @@
 /// Class that initializes the modal view using the passed in campaign information to build the UI.
 internal class FullScreenView: FullView {
 
+    private var statusBarOverlayColor: UIColor {
+        if UIApplication.shared.getCurrentStatusBarStyle() == .lightContent {
+            return UIColor.black.withAlphaComponent(0.4)
+        } else {
+            return UIColor.white.withAlphaComponent(0.4)
+        }
+    }
```

Comment:
You can move it to a UIColor extension:
`private extension UIColor {
    static var statusBarOverlayColor: UIColor {
         if UIApplication.shared.getCurrentStatusBarStyle() == .lightContent {
             return UIColor.black.withAlphaComponent(0.4)
         } else {
             return UIColor.white.withAlphaComponent(0.4)
         }
     }
}`

So in your code, you can do:
`statusBarBackground.backgroundColor = .statusBarOverlayColor`